djangocms_history/models.py

```diff
@@ -284,7 +284,7 @@ class PlaceholderOperation(models.Model):
     operation_type = models.CharField(max_length=30, choices=OPERATION_TYPES)
     token = models.CharField(max_length=120, db_index=True)
     origin = models.CharField(max_length=255, db_index=True)
-    language = models.CharField(max_length=5, choices=settings.LANGUAGES)
+    language = models.CharField(max_length=15, choices=settings.LANGUAGES)
```

Comment:
Looks good but there's another one that stores language.
`PlaceholderAction.language`